[2022-11-27T10:55:05.258+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: hello_world_dag.hello_world_task scheduled__2022-11-27T10:53:00+00:00 [queued]>
[2022-11-27T10:55:05.332+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: hello_world_dag.hello_world_task scheduled__2022-11-27T10:53:00+00:00 [queued]>
[2022-11-27T10:55:05.338+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T10:55:05.339+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 4
[2022-11-27T10:55:05.339+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T10:55:05.413+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): hello_world_task> on 2022-11-27 10:53:00+00:00
[2022-11-27T10:55:05.422+0000] {standard_task_runner.py:55} INFO - Started process 69 to run task
[2022-11-27T10:55:05.489+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task', 'scheduled__2022-11-27T10:53:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/dag_helloworld.py', '--cfg-path', '/tmp/tmpnf9hl78h']
[2022-11-27T10:55:05.498+0000] {standard_task_runner.py:83} INFO - Job 229: Subtask hello_world_task
[2022-11-27T10:55:16.592+0000] {local_task_job.py:82} ERROR - Received SIGTERM. Terminating subprocesses
[2022-11-27T10:55:16.613+0000] {local_task_job.py:159} INFO - Task exited with return code 143
